src/topology/algebra/group.lean

```diff
@@ -940,6 +940,28 @@ local attribute [instance] topological_group.regular_space
 @[to_additive]
 lemma topological_group.t2_space [t1_space G] : t2_space G := regular_space.t2_space G
 
+variables {G} (S : subgroup G) [subgroup.normal S] [hS : is_closed (S : set G)]
+
+include hS
+
+@[to_additive]
+instance subgroup.t1_quotient_of_is_closed :
+  t1_space (G ⧸ S) :=
+begin
+  rw ← quotient_group.ker_mk S at hS,
+  exact topological_group.t1_space (G ⧸ S) ((quotient_map_quotient_mk.is_closed_preimage).mp hS)
+end
+
+@[to_additive]
+instance subgroup.regular_quotient_of_is_closed :
+  regular_space (G ⧸ S) :=
+infer_instance
+
```

Comment:
Could we drop this also if we promoted `topological_group.regular_space` to be an instance above. I.e., drop this and replace:
```lean
lemma topological_group.regular_space [t1_space G] : regular_space G :=
```
above with:
```lean
@[priority 100] -- see Note [lower instance priority]
instance topological_group.regular_space [t1_space G] : regular_space G :=
```

Reply:
The linter complains if I do that, it seems to create a loop

Reply:
Ah right of course, a regular space is t1 so `topological_group.regular_space` cannot be a global instance.